packages/config-array/tests/config-array.test.js

```diff
@@ -2860,6 +3010,30 @@ describe("ConfigArray", () => {
 				);
 			});
 
+			it("should return true when the directory is the parent of the base path", () => {
```

Comment:
This is to ensure that a relative path like `".."` without a slash is recognized as external.